packages/tsconfig/base.json

```diff
@@ -23,6 +24,7 @@
     "skipLibCheck": true,
     "verbatimModuleSyntax": true,
     "tsBuildInfoFile": "${configDir}/node_modules/.cache/tsbuildinfo.json",
-    "noUncheckedSideEffectImports": true
+    "noUncheckedSideEffectImports": true,
+    "allowImportingTsExtensions": true
```

Comment:
why this?

Reply:
To allow importing the config with a `.ts` extension (which is needed for turborepo to cache right).

Reply:
> To allow importing the config

What config?

Reply:
I imported this with a `.ts`, which errored. I must've changed that on accident, but it looks like it works, so 🤷‍♂️

https://github.com/AndreaPontrandolfo/sheriff/blob/96b7ddbce9495d036e8af6f8e5a42a44eea7650f/packages/eslint-config-sheriff/eslint.config.ts#L1-L2